src/PrestaShopBundle/PrestaShopBundle.php

```diff
@@ -60,8 +60,8 @@ public function build(ContainerBuilder $container)
     {
         $container->addCompilerPass(new DynamicRolePass());
         $container->addCompilerPass(new PopulateTranslationProvidersPass());
-        $container->addCompilerPass(new LoadServicesFromModulesPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, 1);
-        $container->addCompilerPass(new LoadServicesFromModulesPass('admin'), PassConfig::TYPE_BEFORE_OPTIMIZATION, 1);
+        $container->addCompilerPass(new LoadServicesFromModulesPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, 200);
```

Comment:
Same comment here to explain the priority, it's just so that a future developer understands why this specific value is used and doesn't change it

Reply:
we could even add the # of the PR so that future devs can see the full explanation if needed

Reply:
I added a link to this PR in the doc comment.
https://github.com/PrestaShop/PrestaShop/pull/30588#discussion_r1050611213